site/content/projects/contenta.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Contenta
+repo: contentacms
+homepage: http://www.contentacms.org
+opensource: "Yes"
+typeofcms: "API Driven"
+supportedgenerators:
+  - All
+description: Contenta is the community driven API distribution for Drupal 8.
+---
+## Contenta is a decoupled Drupal 8 distribution
+
+Contenta is the community driven API distribution for Drupal 8, and provides the robustness and flexibility of Drupal in a friendly API-first package.
+
+It provides a standard platform that is API ready along with demo content and example front-end applications. Contenta intends to ease the pain of using, or simply trying, decoupled Drupal.
```

Comment:
It would be nice to also mention the documentation aspect

Reply:
👍 Done!

Reply:
👍